### PR TITLE
Adds Snyk GitHub action using CLI

### PR DESCRIPTION
**Pull request by SHession**
## What does this change?

Adds Snyk integration using the CLI via GitHub actions using a [new reusable workflow](https://github.com/guardian/.github/pull/11). This should provide us more consistent and accurate results.

## How to test

Does the GitHub action kick off correctly, do the result display as expected in the [Snyk dashboard](https://app.snyk.io/org/guardian/projects)?

### CODE REVIEW

**Comment by SHession** [comment]
Seems to be working as inteded, except it is determining the wrong `"nodeVersion": "v14.17.6"` and saying it has `Encountered multiple node lockfiles files`. Not sure why this is happening but would be interested if anyone has any thoughts. I have left the action in a debug state for reviewers.

File: .github/workflows/snyk.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    schedule:
+        - cron: "0 6 * * *"
+    push:
+        # branches:
+        #     - main
+    workflow_dispatch:
+
+jobs:
+  security:
+    uses: guardian/.github/.github/workflows/sbt-node-snyk.yml@add-generic-snyk-workflow
```
[the remark this answers is not in the document]
Comment (SHession) [reply .github/workflows/snyk.yml:13]: Yeah good point, I would be open to that idea.  I'm not sure if there is an important distinction between `workflows` and `actions` to be made. I think workflows may need to be [referenced in this convoluted way](https://docs.github.com/en/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow).

**Comment by jonathonherbert** [comment]
Ace! I can't see anything in checks – do we need to do anything special to kick this off, or perhaps I'm looking in the wrong place?

**Review by jonathonherbert — APPROVED** [review]
Can see the action [here](https://github.com/guardian/workflow-frontend/actions), and can see a CLI project in [snyk](https://app.snyk.io/org/guardian/project/3953777f-35d4-4cff-a554-3c17273f03cf) 👍

**Pull request merged**